cmd/root_commands.go

```diff
@@ -1,24 +1,30 @@
 package cmd
 
 import (
+	"context"
 	"dp-utils/config"
 	"dp-utils/customisemydata"
 	"dp-utils/out"
+	projectgeneration "dp-utils/project-generation"
+	repository "dp-utils/repository-creation"
 	"dp-utils/zebedee"
+	"github.com/ONSdigital/log.go/log"
 	"math/rand"
 	"os"
 	"path/filepath"
+	"strings"
 	"time"
 
 	"github.com/spf13/cobra"
 )
 
 var (
-	Root     *cobra.Command
-	Version  *cobra.Command
-	Clean    *cobra.Command
-	Import   *cobra.Command
-	Generate *cobra.Command
+	Root       *cobra.Command
+	Version    *cobra.Command
+	Clean      *cobra.Command
+	Import     *cobra.Command
+	Generate   *cobra.Command
+	CreateRepo *cobra.Command
```

Comment:
I've never used cobra, so don't know if this is a quirk of the library, but do these vars actually need to be exported?